test/test.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not sure why this file was created. Maybe because We are using the workspace to build from in travis. I could not find people suggesting to ignore this file, so it is added